tests/test_wsgi.py

```diff
@@ -123,3 +128,40 @@ def appview():
         }
         res = client.get("/", headers=h, environ_base=e)
         assert json.loads(res.get_data(as_text=True)) == data[num_proxies]
+
+
+# Test with type annotations
+def test_wsgi_with_type_annotations():
+    """Test wsgi functions with proper type annotations."""
+    from flask import Flask
+
+    from invenio_base.wsgi import (
+        WSGIApplication,
+        create_wsgi_factory,
+        wsgi_proxyfix,
+    )
```

Comment:
Can we move those to top-level or do they *have* to be inside the function here?